### ISSUE TITLE
Generate apps through script

### ISSUE THREAD

**Issue by vinayvenu** [avniproject/avni-client#888]
### Story
As a developer, I should be able to create apps with different styles with a script

### Developer experience
- make setup [appName] should set up a new app. Once this is done, developer can do the normal workflow. This will clean up any necessary parts of the old app and set up the new app

### Tasks
- Directory android-template as template for any common content
- Directory android-apps for app-specific content
- MainApplication.java and MainActivity.java to stay in android-template and modified based on need
- Restart package to go to a different package org.avni but stay within android-template
- res folder goes to android-apps/appName
- build.gradle stays in android-template
- keystores to be renamed to common name (app-release-keystore) and kept within android-apps/appName
- google-services.json to stay within android-apps/appName
- avni-logo.png to be renamed to app-logo.png and stay within android-apps/appName
- prod.json to be renamed to avni.json
- AndroidManifest.xml to stay in android-template and modified to match app name and package names
- Script should work on atleast mac and linux
- File in android-apps/appName to specify app name, package name and server url
- Files in android-apps/appName for logo and keystores
- Standard way to specify keystore passwords for each app name that will be picked up from the environment
- Verification mechanism to ensure all of this is present
- Ability to generate both apk and app bundle (we will move to app bundles as part of this). app bundles will be mandatory for lfe, but we can move Avni as well to app bundles so they are equivalent
- Both LFE and Avni needs to be uploaded to Playstore (alpha channel) to confirm everything works
- CI does not break

### Questions
- Shell scripts.. 

### Implementation comments:
- The above approach not used and instead android flavoring concept used.
- Firebase analytics for LFE was already not working. So didn't fix it as part of this card.
- During release these needs to be made sure it is working as expected: app signing, bundle version code, and need to remove bugsnag test log
- supporting upload translations for multiple flavours for webapp - out of scope

### Out of Scope
- CI

**Comment by petmongrels** [comment]
Documentation.
- specify which build.gradle
- no command to release a non-default flavor

**Comment by vinayvenu** [comment]
The implementation uses Android flavors, which is different from the approach described in the story.
